x-pack/plugins/reporting/server/export_types/csv/server/generate_csv/hit_iterator.ts

```diff
@@ -55,10 +58,10 @@ export function createHitIterator(logger: LevelLogger) {
       );
     }
 
-    function scroll(scrollId: string | undefined) {
+    async function scroll(scrollId: string | undefined) {
       logger.debug('executing scroll request');
       return parseResponse(
-        callEndpoint('scroll', {
+        await callEndpoint('scroll', {
```

Comment:
The previous code was passing through a promise, for it to be resolved in some other context